tests/storage/memorydump.go

```diff
@@ -598,119 +437,17 @@ var _ = SIGDescribe("Memory dump", func() {
-			Eventually(func() error {
-				return memorydumpCommand()
-			}, 20*time.Second, 2*time.Second).ShouldNot(HaveOccurred())
-		}
-
-		memoryDumpVirtctlGetWithDownload := func(name, namespace, claimName, outputFile string) {
-			By("Invoking virtctl memory dump")
-			commandAndArgs := []string{commandMemoryDump, "get", name, virtCtlNamespace, namespace}
-			commandAndArgs = append(commandAndArgs, fmt.Sprintf(virtCtlClaimName, claimName))
-			commandAndArgs = append(commandAndArgs, fmt.Sprintf(virtCtlOutputFile, outputFile))
-			memorydumpCommand := clientcmd.NewRepeatableVirtctlCommand(commandAndArgs...)
-			Eventually(func() error {
-				return memorydumpCommand()
-			}, 20*time.Second, 2*time.Second).ShouldNot(HaveOccurred())
-		}
-
-		memoryDumpVirtctlCreateWithDownload := func(name, namespace, claimName, outputFile string) {
-			By("Invoking virtctl memory dump with create flag")
-			commandAndArgs := []string{commandMemoryDump, "get", name, virtCtlNamespace, namespace}
-			commandAndArgs = append(commandAndArgs, fmt.Sprintf(virtCtlClaimName, claimName))
-			commandAndArgs = append(commandAndArgs, virtCtlCreate)
-			commandAndArgs = append(commandAndArgs, fmt.Sprintf(virtCtlOutputFile, outputFile))
-			if !checks.IsOpenShift() {
-				targetPort := fmt.Sprintf("%d", 37548+rand.Intn(6000))
-				commandAndArgs = append(commandAndArgs, virtCtlPortForward, fmt.Sprintf(virtCtlLocalPort, targetPort))
-			}
-			memorydumpCommand := clientcmd.NewRepeatableVirtctlCommand(commandAndArgs...)
-			Eventually(func() error {
-				err := memorydumpCommand()
-				if err != nil {
-					_, getErr := virtClient.CoreV1().PersistentVolumeClaims(namespace).Get(context.Background(), claimName, metav1.GetOptions{})
-					if getErr == nil {
-						// already created the pvc can't call the memory dump command with
-						// create-claim flag again
-						By("Error memory dump command after claim created")
-						memoryDumpVirtctlGetWithDownload(name, namespace, claimName, outputFile)
-						return nil
-					}
-				}
-				return err
-			}, 20*time.Second, 2*time.Second).ShouldNot(HaveOccurred())
-		}
-
-		BeforeEach(func() {
-			sc, exists := libstorage.GetRWOFileSystemStorageClass()
-			if !exists {
-				Skip("Skip no filesystem storage class available")
-			}
-			libstorage.CheckNoProvisionerStorageClassPVs(sc, numPVs)
-
-			vm = createAndStartVM()
-			outputFile = fmt.Sprintf(defaultOutput, rand.String(12))
-		})
-
-		AfterEach(func() {
-			if vm != nil {
-				deleteVirtualMachine(vm)
-			}
-			pvc, err := virtClient.CoreV1().PersistentVolumeClaims(testsuite.NamespaceTestDefault).Get(context.Background(), memoryDumpPVCName, metav1.GetOptions{})
-			if err == nil && pvc != nil {
-				deletePVC(pvc)
-			}
-			if err := os.Remove(outputFile); err != nil && !goerrors.Is(err, os.ErrNotExist) {
-				Fail(err.Error())
-			}
-		})
-
-		It("[test_id:9344]should create memory dump and download it", func() {
```

Comment:
and this one should also test the enablement of downloading a memory dump

Reply:
That is covered by this test, which was enhanced during the cleanup.

https://github.com/kubevirt/kubevirt/blob/715b28f03422da3d930afce38bc90a3e657e7cf0/pkg/virtctl/memorydump/memorydump_test.go#L411